packages/next-plugin-sentry/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "@sentry/next-plugin-sentry",
+  "version": "6.2.1",
```

```suggestion
  "version": "6.2.5",
```
We need to sync version with `master`